store/sqlstore/post_store.go

```diff
@@ -1321,7 +1321,7 @@ func (s *SqlPostStore) determineMaxPostSize() int {
 				table_name = 'posts'
 			AND	column_name = 'message'
 		`); err != nil {
-			mlog.Error(utils.T("store.sql_post.query_max_post_size.error") + err.Error())
+			mlog.Error(utils.T("store.sql_post.query_max_post_size.error"), mlog.Err(err))
```

Comment:
Not related to this PR, but we don't translate log messages, so probably makes sense to remove the `uitls.T` call here and use directly the english version of the translated string.

Reply:
done